todo_app/app.py

```diff
@@ -1,29 +1,40 @@
 from flask import Flask
 from flask import render_template
 from flask import request, redirect, url_for
+import os
 import todo_app.data.session_items
+import todo_app.data.trello_items
+import json
+import requests
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    todo_items = todo_app.data.session_items.get_items()
+    todo_items = todo_app.data.trello_items.get_items()
     return render_template('index.html', todo_items=todo_items)
 
 @app.route('/add', methods = ['POST', 'GET'])
 def add_item():
     if request.method == 'POST':
         new_todo = request.form.get('todo_item')
-        todo_app.data.session_items.add_item(new_todo)
+        todo_app.data.trello_items.add_item(new_todo)
```

Comment:
It's nice how clean this swap out is; a good sign you've kept the interfaces consistent!